plutus-core/plutus-core/test/TypeSynthesis/Spec.hs

```diff
@@ -126,29 +125,28 @@ test_typecheckIllTyped =
                 TypeErrorE (NameMismatch {}) -> True
                 _                            -> False
             ]
-
 test_typecheckAllFun
-    :: forall fun. (ToBuiltinMeaning DefaultUni fun, Show fun)
+    :: forall fun. (ToBuiltinMeaning DefaultUni fun, Show fun, Show (BuiltinSemanticsVariant fun))
     => String
     -> BuiltinSemanticsVariant fun
-    -> TestTree
-test_typecheckAllFun name semvar
-    = runTestNestedIn ["plutus-core", "test", "TypeSynthesis", "Golden"]
-    . testNested name
+    -> TestNested
+test_typecheckAllFun name semVar
+    = testNestedNamed name (show semVar)
```

Comment:
This would previously print `DefaultFun` three times (once per semantics variant) as if the same test was running three times in a row. Now we can see that it's a different test each time by printing the semantics variant.